Instructor-Guide--Users.md

```diff
@@ -51,8 +52,8 @@ A count of the number of grace credits a student has left can be seen in the "Gr
 
 Three actions other than assigning grace credits may be performed from the drop down menu:
 
-- **Add to section**: This allows you to add the selected student(s) to the specified lecture section.
-![Add to section](images/users-add-to-section.png)
+- **Update section**: This allows you to update the lecture section for the selected student(s). The available options are a "no section" - if you want the student to (temporarily) not be in any section - and the existing lecture sections for the course.
```

Comment:
Let's replace "lecture section" with just "section" (since there might be other uses of sections beyond lectures).

Similarly, don't assume that students will only temporarily not be in a section. Reword the part explaining "no section" to something like "if you want to remove the students from their current section".